### ISSUE TITLE
Invalid enum values in watch window vs memory view (MinGW-w64 + cv2pdb)

### ISSUE THREAD

**Issue by McKay42** [x13pixels/remedybg-issues#256]
Toolchain: [x86_64-8.1.0-release-posix-seh-rt_v6-rev0.7z](https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/8.1.0/threads-posix/seh/) and [cv2pdb-0.51.zip](https://github.com/rainers/cv2pdb/releases/tag/v0.51)

main.cpp:
```
enum TYPE : int
{
	INVALID = -1,
	
	FIRST_TYPE = 1,
	SECOND_TYPE = 2,
	THIRD_TYPE = 3,
};

int main(void)
{
	TYPE shouldBeFirstType = TYPE::FIRST_TYPE;
	int iShouldBeFirstType = shouldBeFirstType;
	
	TYPE shouldBeSecondType = TYPE::SECOND_TYPE;
	int iShouldBeSecondType = shouldBeSecondType;
	
	TYPE shouldBeThirdType = TYPE::THIRD_TYPE;
	int iShouldBeThirdType = shouldBeThirdType;
	
	return 0;
}
```

build.bat (note https://github.com/x13pixels/remedybg-issues/issues/247):
```
g++ -g3 -O0 -o example.exe main.cpp
cv2pdb example.exe
remedybg example.exe
```

<img width="718" alt="Capture" src="https://user-images.githubusercontent.com/16856443/178998749-eebb97fb-46c1-4822-9945-3ec545d5cc7b.PNG">

Basically all enum values everywhere are always shown incorrectly in the watch window, that also includes enum classes and anonymous/inner enums with or without specified types. They always show as 0. Jumping to the memory view shows the correct value there, same as casting/static_cast-ing to a temporary.

Not sure if this is a cv2pdb bug or not, probably another edge case?

**Comment by x13pixels** [comment]
[0.3.8.0] Now look past any modifiers of enum's underlying type (const/volatile).
